core/templates/components/forms/custom-forms-directives/object-editor.directive.ts

```diff
@@ -57,14 +65,29 @@ angular.module('oppia').directive('objectEditor', [
         scope.getIsEditable = function() {
           return scope.isEditable;
         };
+        scope.updateValue = function(e) {
+          scope.value = e;
+          $rootScope.$applyAsync();
```

Comment:
Why we need rootScope here? (Just wanted to know, in can this can be helpful for other developers.)